addons/hr/static/src/thread_service_patch.js

```diff
@@ -51,6 +51,10 @@ const threadServicePatch = {
         }
         return _super({ userId: employee.user_id });
     },
+    async getEmployeeIdFromPartnerId(partnerId) {
+        const employees = await this.orm.silent.searchRead("hr.employee", [["user_partner_id", "=", partnerId]], ["id"])
+        return employees[0]?.id;
+    },
```

Comment:
Indirection `getEmployeeIdFromPartnerId` to thread service patch is useless: can simply call this code in the thread action `setup`.